README.md

```diff
@@ -11,6 +11,7 @@ pip install -r requirements.txt
 set FLASK_APP=application
 flask run
 ```
+Please note, one of the pre-requisites is pg_config which is installed by installing the PostGresSQL client
```

Comment:
Looks like this is specific to Mac/linux. Based on [stack overflow](https://stackoverflow.com/a/46877364/3088653) installing the binary package should solve this `pip install psycopg2-binary`